README.md

```diff
@@ -1,187 +1,130 @@
-
-Once javascript is compiled with `./do compile:javascript`, your module will be placed into a bundle. Including that bundle in a webpage will give provide you access to your module.
-
-## Handlebars (`views/*.hbs`)
-
-```html
-<!-- use the `templates` block -->
-<% block templates %>
-  <!-- include a .hbs template -->
-  <%= partial('my_template_1', 'form/templates/toolbar/fields.hbs') %>
-
-  <!-- include a .hbs template and register it as a partial -->
-  <%= partial('my_template_2', 'form/templates/blocks.hbs', '_my_partial') %>
-
-  <!-- custom partial using partial defined above -->
-  <script id="my_template_3" type="text/x-handlebars-template">
-    {{> _my_partial }}
-  </script>
-<% endblock %>
-```
-
-# i18n
-- Use the regular WordPress functions in PHP and Twig:
-
-```php
-__()
-_n()
-_x()
-```
-
-```html
-<p>
-  <%= __('Click %shere%s!') | format('<a href="#">', '</a>') | raw %>
-</p>
-```
-
-```html
-<p>
-  <%= _n('deleted %d message', 'deleted %d messages', count) | format(count) %>
-  <!-- count === 1 -> "deleted 1 message" -->
-  <!-- count > 1 -> "deleted $count messages" -->
-</p>
-```
-
-- Handlebars.
-
-You can use Twig i18n functions in Handlebars, just load your template from a Twig view.
-
-# Build
-
-To build a plugin , run `./build.sh`.
-
-Some build process steps are described below (their dependencies etc.).
-
-## packtranslations step
-
-This step imports translations from Transifex and generates MO files. It requires:
-* `tx` client: https://docs.transifex.com/client/installing-the-client
-* `msgfmt` command (from Gettext package)
-Finally , a `WP_TRANSIFEX_API_TOKEN` environment variable should be initialized with a valid key.
-
-# Publish
-
-The `publish` command currently does the following:
-* Pushes translations POT file to Transifex;
-* Publishes the release in SVN.
-
-Before you run it, you need to:
-1. Ensure there is an up-to-date local copy of MailPoet SVN repository in `.mp_svn` directory by running `./do svn:checkout`.
-2. Have all your features merged in Git `master`, your `mailpoet.php` and `readme.txt` tagged with a new version.
-3. Run `./build.sh` to produce a `mailpoet.zip` distributable archive.
-
-Everything's ready? Then run `./do publish`.
-If the job goes fine, you'll get a message like this:
-```
-Go to '.mp_svn' and run 'svn ci -m "Release 3.0.0-beta.9"' to publish the
-release
-
-Run 'svn copy ...' to tag the release
-```
-It's quite literal: you can review the changes to be pushed and if you're satisfied, run the suggested command to finish the release publishing process.
-
-If you're confident, execute `./do publish --force` and your release will be published to the remote SVN repository without manual intervention (automatically). For easier authentication you might want to set `WP_SVN_USERNAME` and `WP_SVN_PASSWORD` environment variables.
-
-# Acceptance testing
+# Frameworks and libraries
+
```

Comment:
It is hard to pick important libraries and don't list all of them :). 
I would add also these to the list:
twig: templates rendering
monolog: logging
handlebars: templates for editor